### ISSUE TITLE
Heading text checker ignores aria-labels on links

### ISSUE THREAD

**Issue by cbirdsong** [itmaybejj/editoria11y-wp#10]
The heading text checker is flagging the following as blank:

```
<h1 class="header-logo">
  <a href="/" aria-label="Site Name">
    <svg width="303" height="158" viewBox="0 0 303 158" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
       <!-- excluded for brevity -->
    </svg>
  </a>
</h1>
```

Chrome dev tools and VoiceOver both pick up the aria label as the heading's text.

**Comment by itmaybejj** [comment]
Yep; go for the exclusion method then.

Luckily #8 will be a quick fix. Hopefully within the next week or so.

**Comment by itmaybejj** [comment]
Fixed in [1.0.13!](https://github.com/itmaybejj/editoria11y-wp/releases/tag/1.0.13)
